src/app/docs/commands/blob-list-blobs/page.mdx

```diff
@@ -12,17 +12,15 @@ export const metadata = {
     ]} />
 </div>
 
-# blob list blobs 
+# blob list collections
```

Comment:
This is the `blob-list-blobs` page according to the file name, there is a second `blob-list-collections` page. Should they both exist?